src/mlpack/methods/ann/layer/concat.hpp

```diff
@@ -53,6 +53,19 @@ class Concat
   Concat(const bool model = false,
          const bool run = true);
 
+  /**
+   * Create the Concat object using the specified parameters.
+   *
+   * @param inputSize A vector denoting input size of each layer added.
+   * @param axis Concat axis.
+   * @param model Expose all network modules.
+   * @param run Call the Forward/Backward method before the output is merged.
+   */
+  Concat(arma::Row<size_t> inputSize,
```

Comment:
Should this be `const arma::Row<size_t>& inputSize`?

Reply:
Actually, I need it as `arma::Row<size_t> inputSize` as a need to pass the values using initialization list `Concat<>({x,y,z},axis) ` .